psydac/linalg/stencil.py

```diff
@@ -893,6 +893,24 @@ def dot( self, v, out=None):
         return out
 
     def vdot( self, v, out=None):
+        """
+        Return the matrix/vector product between the conjugate of self and v.
+        This function optimized this product.
+
+        Parameters
+        ----------
+        v   : StencilVector
+            Vector of the domain of self needed for the Matrix/Vector product
+
+        out : StencilVector
+            Vector of the codomain of self
+
+        Return
```

Comment:
This should be called `Returns`: https://numpydoc.readthedocs.io/en/latest/format.html#returns.

Reply:
[aca6888](https://github.com/pyccel/psydac/pull/273/commits/aca6888978363382c8312f28c06d83806f1fc8f9)